apps/address-book/test/address-book.js

```diff
@@ -95,14 +95,14 @@ contract('AddressBook App', accounts => {
     })
   })
   context('invalid operations', () => {
-    let [borg, jeanluc] = accounts.splice(1, 2)
+    let [ borg, jeanluc ] = accounts.splice(1, 2)
     before(async () => {
       app.addEntry(borg, 'Borg', 'Individual')
     })
 
     it('should revert when adding duplicate address', async () => {
       assertRevert(async () => {
-        await app.addEnty(borg, 'Burg', 'N/A')
```

Comment:
This makes me think at some point it might be good to do more specific revert handling now that there are revert messages?

Reply:
I now _always_ check that the revert message is as expected before wrapping assertrevert around a transaction attempt to ensure that the revert is happening for the expected reason